lib/jira/commands/tickets.rb

```diff
@@ -36,9 +39,9 @@ def rows
         metadata['issues'].map do |issue|
           [
             issue['key'],
-            issue['fields']['assignee']['name'] || 'Unassigned',
-            issue['fields']['status']['name'] || 'Unknown',
-            truncate(issue['fields']['summary'], 45)
+            (issue['fields']['assignee']['name'] unless issue['fields']['assignee'].nil?) || 'Unassigned',
```

Comment:
can we change this to:
`issue['fields'].fetch('assignee',{})['name'] || 'Unassigned'`
same with the below status conditional